modules/gdscript/tests/scripts/analyzer/errors/typed_constructor_not_implemented.out

```diff
@@ -0,0 +1,2 @@
+GDTEST_ANALYZER_ERROR
+No constructor of "float" matches the signature "float(Array, int, StringName, Variant)"
```

Comment:
That's a confusing message, since this constructor format was not actually typed by the user. This should say only arrays can use this type of constructor since it's the only type that actually have type argument. If/when this we add more possibilities in the future we can change the message.